app/src/main/res/layout/login_register_fragment.xml

```diff
@@ -0,0 +1,210 @@
+                android:layout_height="50dp"
+                android:background="@color/black"
+                android:gravity="center"
+                android:layout_alignParentBottom="true"
+                android:text="@string/create_user"
+                android:textAllCaps="true"
+                android:textColor="@color/warm_grey"
+                android:textSize="18sp"
+                />
+
+            <LinearLayout
+                android:layout_width="match_parent"
+                android:layout_height="match_parent"
+                android:gravity="center_vertical"
+                android:layout_above="@id/create_login_textview"
+                android:layout_below="@id/daydreaming_image_view"
+                android:orientation="vertical"
+                >
+
+                <LinearLayout
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:layout_marginEnd="40dp"
+                    android:layout_marginStart="30dp"
+                    android:orientation="horizontal"
+                    >
+
+                    <TextView
+                        android:id="@+id/register_text_view"
+                        android:layout_width="0dp"
+                        android:layout_height="wrap_content"
+                        android:gravity="center"
+                        android:layout_weight="1"
+                        android:text="@string/register"
+                        android:textAllCaps="true"
+                        android:textColor="@color/pale_teal"
+                        android:textSize="18sp"
+                        android:textStyle="bold"
+                        />
+
+                    <TextView
+                        android:id="@+id/login_text_view"
+                        android:layout_width="0dp"
+                        android:layout_height="wrap_content"
+                        android:gravity="center"
+                        android:layout_weight="1"
+                        android:text="@string/log_in"
+                        android:textAllCaps="true"
+                        android:textColor="@color/white"
+                        android:textSize="18sp"
+                        android:textStyle="bold"
+                        />
+
+                </LinearLayout>
+
+                <RelativeLayout
+                    android:layout_width="match_parent"
+                    android:layout_height="60dp"
+                    android:layout_marginTop="20dp"
+                    >
+
+                    <View
+                        android:layout_width="match_parent"
+                        android:layout_height="match_parent"
+                        android:alpha="0.4"
+                        android:background="@color/black"
+                        />
+
+                    <EditText
+                        android:id="@+id/username_field"
```

Comment:
You could have aligned it with the start of the username_error imageview. 

I know that the image won't be shown until it shows an error, but when it does, will this be shown above the EditText?

Setting big margins for this is like cheating :P 
You could just set the imageView state to `hidden`, not `gone` and that's it.

Adding the cross to the `error_text_view` as a startDrawable, would've been an alternative, which would have saved lines. (Obviously, this is without taking design desitions into account)

Same thing applies for other layouts

For example:

        <ImageView
            android:id="@+id/holy_canoli"
            android:layout_width="wrap_content"
            android:layout_height="match_parent"
            android:background="@color/blueDark"
            android:layout_alignParentEnd="true"
            android:src="@drawable/alerts_tutorial_bf"
            />

        <EditText
            android:layout_width="wrap_content"
            android:layout_height="match_parent"
            android:background="@color/cpb_red"
            android:layout_alignParentStart="true"
            android:layout_toStartOf="@id/holy_canoli"
            />